opm/parser/eclipse/EclipseState/Schedule/Well.hpp

```diff
@@ -164,6 +182,13 @@ namespace Opm {
         const Events& getEvents() const;
         void addEvent(ScheduleEvents::Events event, size_t reportStep);
         bool hasEvent(uint64_t eventMask, size_t reportStep) const;
+        void handleCOMPLUMP(const DeckRecord& record, size_t time_step);
+        void handleCOMPDAT(size_t time_step, const DeckRecord& record, const EclipseGrid& grid, const Eclipse3DProperties& eclipseProperties);
+        void handleCOMPSEGS(const DeckKeyword& keyword, size_t time_step);
+        void handleWELOPEN(const DeckRecord& record, size_t time_step, WellCompletion::StateEnum status);
+        void handleWPIMULT(const DeckRecord& record, size_t time_step);
+        void handleWELSEGS(const DeckKeyword& keyword, size_t time_step);
+
```

Comment:
Moving the "handling" of the keywords from schedule to well seems like a good move. I haven't look into the details, but since there is good test coverage for this part of the code, I am confident that the new code does the same as the old code.